seedlet/assets/sass/blocks/navigation/_editor.scss

```diff
@@ -16,23 +20,33 @@
         }
     }
 
+    .wp-block-navigation-item {
+        > a {
+            border: 0;
```

Comment:
Is this going to play nicely with https://github.com/Automattic/themes/pull/5195 ?

Reply:
Hmm.. possibly, although it looks like the navigation may need to override the `text-decoration` styles instead of the `border`.